src/qt/rpcconsole.h

```diff
@@ -106,7 +106,7 @@ public Q_SLOTS:
     /** Set network state shown in the UI */
     void setNetworkActive(bool networkActive);
     /** Set number of masternodes shown in the UI */
-    void setMasternodeCount(const QString &strMasternodes);
+    void setMasternodeCount();
```

Comment:
it's more like `updateMasternodeCount` now, as we're not setting it to an external value anymore